nnvm/tests/python/compiler/test_top_level1.py

```diff
@@ -365,6 +365,129 @@ def forward(x):
     inputs = [('x', (1, 3, 28, 28), x)]
     helper(y, inputs, dtype, forward)
 
+def verify_lrn(n, c, h, w, size, axis, bias, alpha, beta):
```

Comment:
`def verify_lrn(ishape, size, axis, bias, alpha, beta)` and using ishape instead of dshape looks simpler.